sdl-core/src/main/scala/io/smartdatalake/util/evolution/SchemaEvolution.scala

```diff
@@ -173,17 +186,19 @@ object SchemaEvolution extends SmartDataLakeLogger {
       // this defines the ordering of the resulting DataFrame's
       val tgtCols = if (Environment.schemaEvolutionNewColumnsLast) {
         // new columns last
-        oldColsWithoutTechCols ++ newColumns(oldDf, newDf) ++ colsToIgnore
+        if (caseSensitiveComparison) oldColsWithoutTechCols ++ newColumns(oldDf, newDf) ++ colsToIgnore
+        else oldColsWithoutTechCols ++ newColumns(oldDf, newDf) ++ colsToIgnore.map(_.toLowerCase)
```

Comment:
```suggestion
        if (caseSensitiveComparison) oldColsWithoutTechCols ++ newColumns(oldDf, newDf) ++ (if (caseSensitiveComparison) colsToIgnore else colsToIgnore.map(_.toLowerCase))
```

Reply:
Fixed in Commit cb32221f47394f53c4b06c5d6b2c277df14e2d91